test/parallel/test-whatwg-url-tojson.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const URL = require('url').URL;
+
+// Tests below are not from WPT.
+const url = new URL('http://example.org');
```

Comment:
Also this should be easier if we are updating a file instead of adding a new one(no need to do the conversion, just copy-paste)